### ISSUE TITLE
Can lassonet works in transformer or S4(Structured State Spaces)?

### ISSUE THREAD

**Issue by xueliu8617112** [lasso-net/lassonet#38]
i hope to use lassonet in S4, is it okay to do that?

**Comment by louisabraham** [comment]
Are you talking about Structured State Space for Sequence Modeling? I don't know very well the details nor how you want to apply LassoNet.

Do you have a concrete example in mind?

**Comment by xueliu8617112** [comment]
yeah, it is S4 for Structured State Space for Sequence Modeling. I just want to do some time series prediction work in the industry. I see u use the NN or CNN or AE as the backbone, so I wonder whether s4 can be utilized as the backbone.

**Comment by louisabraham** [comment]
CNN as backbone is very experimental. The goal of LassoNet is to select variables in a meaningful space (eg features).

**Comment by 943fansi** [comment]
@xueliu8617112 I am also interested in this topic. May I ask if you have made any substantial progress regarding this issue recently?

**Comment by ShailenSmith** [comment]
Hi @louisabraham, I'm a PhD student working on a project where we're hoping to use an extension of LassoNet for convolutional layers. The readme mentions that adding support for convolutional layers is in the works -- do you have any updates on this, or ideas for how the implementation would be done?

**Comment by louisabraham** [comment]
Hi, I should update the readme, we are not working on this anymore.

The difficulty is that you want to select features that are interpretable!
